src/sql/engine/px/ob_dfo_scheduler.cpp

```diff
@@ -463,6 +463,14 @@ int ObSerialDfoScheduler::dispatch_sqcs(ObExecContext &exec_ctx,
                               coord_info_.interrupt_id_,
                               &sqc);
         args.set_serialize_param(exec_ctx, const_cast<ObOpSpec &>(*dfo.get_root_op_spec()), *phy_plan);
+        // 设置sqc的record_data_字段
+        ObQcRecordData& record_data = sqc.get_record_data();
```

Comment:
Why do you init sqc data in dispatch_sqc()?  Why not init these value earlier in SQC data structure initialization phase?

Reply:
> Why do you init sqc data in dispatch_sqc()? Why not init these value earlier in SQC data structure initialization phase?

I have put it to `ObPXServerAddrUtil::alloc_by_XXX_distribution` when ObPxSqcMeta is initializing